charts/weave-policy-agent/values.yaml

```diff
@@ -42,3 +42,7 @@ policySource:
   # branch:
   # path: ./  # Could be a path to the policies dir or a kustomization.yaml file
   # secretRef: policy-library-auth  # (Optional): Name of the K8s secret with private repo auth credentials
+  # sourceRef: # Could specify a name for an existing GitSource reference instead of creating a new one
+  #   kind: GitRepository
+  #   name: policy-library
+  #   namespace: "{{ .Release.Namespace }}"
```

Comment:
This should be a fixed value not a template value

Reply:
i am not sure what the fixed value is, didn't we agree to set the namespace the same as the release namespace?

Reply:
You can't use template values in the `values` file as far as I know. We can use a value like `flux-system`

Reply:
no you could use it as long las you're using `tpl` function in the template

Reply:
i prefer to make it a placeholder, for example :
```yaml
  #   name: < source name >
  #   namespace:  < source namespace >
```

Reply:
@alichaddad @ahsayde but as i remember we decided to set the default values here

Reply:
We already have a default value for the name and kind and it makes sense to follow that in the namespace, using a template value makes it a bit confusing

Reply:
okay so will use `flux-system` as the default. @ahsayde any concerns?

Reply:
@nohasaayed no it's ok for me